docs/system-administration/DNS/debian-stretch.md

```diff
@@ -0,0 +1,721 @@
+---
+title: Debian (Stretch / Buster) DNS Configuration.
+summary: A guide on how to setup Primary and Secondary caching servers with zone authority under Debian (stretch and Buster).
+date: 2020-11-26
+authors:
+  - Adam Spann
+---
+
+## A note of warning!
+  This guide is only intended for those that plan to use these servers within their own networks running behind a NAT router. Or with the service only available on the internal interfaces. This configure is not intended for WAN (Wide Area Network) interfaces.
+
```

Comment:
Needs an exposition - what is the purpose of this page and what do you hope to communicate with it?

Reply:
I believe this has been done in my latest update.